Sources/LibMobileCoin/McConstants.swift

```diff
@@ -74,7 +74,13 @@ extension McConstants {
     static let DEFAULT_SUBADDRESS_INDEX: UInt64 = 0
 
     /// An account's "default change address" is its first subaddress.
-    static let DEFAULT_CHANGE_SUBADDRESS_INDEX: UInt64 = 1
+    static let DEFAULT_CHANGE_SUBADDRESS_INDEX = UInt64.max - 1
```

Comment:
Just wondering why we do max-1 instead of just max?  Is that in a spec somewhere?

Reply:
https://github.com/mobilecoinfoundation/mcips/pull/36